UI/src/model/selectedMassTransitStop.js

```diff
@@ -213,7 +241,7 @@
       var isRequiredProperty = function(publicId) {
         //ignore if it is a terminal
         //TODO we need to get a way to know the mandatory fields depending on the bus stop type (this was code after merging)
-        if(currentAsset.stopTypes && currentAsset.stopTypes[0] == 6 && _.some())
+        if(currentAsset.stopTypes && ( currentAsset.stopTypes[0] == 6 || currentAsset.stopTypes[0] == 7 ) && _.some())
```

Comment:
What does that _.some() do? Alone it seems to always return false
Also, why are we checking for 'liitetyt_pysakit' == publicId for type 7 stops? I think it is only used for terminals

Reply:
@oskarmalmstrom the  _.some() was there before. it has not added in this commit.
About the  'liitetyt_pysakit' == publicId for type 7 is for service points.